pkg/app/piped/trigger/trigger.go

```diff
@@ -287,10 +287,6 @@ func (t *Trigger) checkRepoCandidates(ctx context.Context, repoID string, cs []c
 			deploymentChainID = c.command.GetChainSyncApplication().DeploymentChainId
 			deploymentChainBlockIndex = c.command.GetChainSyncApplication().BlockIndex
 
-		case model.TriggerKind_ON_OUT_OF_SYNC:
-			strategy = model.SyncStrategy_QUICK_SYNC
-			strategySummary = "Quick sync to attempt to resolve the detected configuration drift"
-
```

Comment:
I think we still need this strategySummary anyway, so how about this

```go
		case model.TriggerKind_ON_OUT_OF_SYNC:
			strategy = model.SyncStrategy_AUTO
			strategySummary = "Sync to attempt to resolve the detected configuration drift"
```

Reply:
strategySummary is unused when SyncStrategy is AUTO.
This is the same logic as with OnCommit trigger.

Reply:
I got it, you're right 👍